### PR TITLE
docs(utils) document updates random_string return

### PR DESCRIPTION
**Pull request by p0pr0ck5**

### CODE REVIEW

**Comment by nijikokun** [comment]
Looks good to me, would suggest using `The random string (modified ...` to `A modified ...`

**Comment by p0pr0ck5** [comment]
@nijikokun thanks, done.

File: app/docs/0.10.x/lua-reference/modules/kong.tools.utils.html [hunk 1]
```diff
@@ -349,7 +349,7 @@ <h2 class="section-header "><a name="Functions">Functions</a></h2>
     <h5>Returns:</h5>
     <ul>
       <li>
-        string  The random string (a uuid without hyphens)
+        string  The random string (a modified Base64-encoded psuedorandom data generated via CSPRNG).
```
Comment (thibaultcha) [inline app/docs/0.10.x/lua-reference/modules/kong.tools.utils.html]: "psuedorandom"

**Comment by Tieske** [comment]
unless I'm mistaken this is generated from the source code using Ldoc, so the proper place to update would be in the source code and then regenerate.

**Comment by p0pr0ck5** [comment]
ah thanks @Tieske :)

**Comment by Tieske** [comment]
@p0pr0ck5 can the branch be deleted?

**Pull request closed without merging**